src/Apps/gEvGen.cxx

```diff
@@ -750,23 +724,13 @@ void GetCommandLineArgs(int argc, char ** argv)
     exit(1);
   }
 
-  // neutrino PDG code
-  if( parser.OptionExists('p') ) {
-    LOG("gevgen", pINFO) << "Reading neutrino PDG code";
-    gOptNuPdgCode = parser.ArgAsInt('p');
-  } else {
-    LOG("gevgen", pFATAL) << "Unspecified neutrino PDG code - Exiting";
```

Comment:
Probably want to keep this section with the error note that one _must_ specify the probe PDG.  Or at least handle it later that the probe PDG is unset as there's no default.